src/merge_duplicate_usernames.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module is a work in progress intended to fix duplicate username.
+
+Issues from pydriller.
+"""
+
+
+# NOTE not used yet, has no impact
+def delete_duplicates(data, keys_to_delete):
+    """Delete keys from dictionary, keys are sent in a list."""
+    dictionary = data
+    for key in keys_to_delete:
+        print(dictionary[key], " will be removed")
+        del dictionary[key]
+        print(key in dictionary)
+    return dictionary
+
+
+# NOTE not used yet, has no impact
+def parse_email(email):
+    """Locate @ and + sign in email and returns the string between them."""
+    # find the index of @ sign
+    plus_index = email.find("+")
+    # find the index of + sign
+    at_index = email.find("@")
+    # slices string accordingly
+    revised_email = email[(plus_index + 1) : at_index]
+    return revised_email
```

Comment:
An easier way is just to split the string with delimiters like "@" and "+"

Reply:
Thank you for suggesting that. I will keep that in mind. However, we might use a python package in the future that would facilitate this whole process.